src/Spago/PackageSet.hs

```diff
@@ -202,16 +204,26 @@ isRemoteFrozen :: Dhall.Import -> [Bool]
 isRemoteFrozen (Dhall.Import
   { importHashed = Dhall.ImportHashed
     { importType = Dhall.Remote _
+    , hash
     , ..
     }
   , ..
   })             = [isJust hash]
 isRemoteFrozen _ = []
 
 
+localImportPath :: Dhall.Import -> Maybe System.IO.FilePath
+localImportPath (Dhall.Import
+  { importHashed = Dhall.ImportHashed
+    { importType = Dhall.Local Dhall.Here file
+    }
+  })              = Just $ Text.unpack $ "." <> pretty file
```

Comment:
Did you mean this?

```suggestion
  })              = Just $ Text.unpack $ "." </> pretty file
```

If not, I'm not sure why we need to concatenate a `.` 🤔

Reply:
`pretty file` returns a filepath like `/../my-package/spago.dhall`, with a leading `/`. I guess we could drop that instead of prepending the `.`?

Reply:
Right. How about calling `pretty` on the `importType`? [Its `Pretty` instance already does what we're doing here](https://github.com/dhall-lang/dhall-haskell/blob/640913f8d2cff6d80dfa836efa45ba3aa4220545/dhall/src/Dhall/Core.hs#L276)

Reply:
That'll do nicely!